### ISSUE TITLE
Coordinates not showing up in "My Recent Uploads" picture

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#1839]
A person uploaded https://commons.wikimedia.org/wiki/File:2018-07-11,_Neubau_der_Freiburger_Kronenbr%C3%BCcke_(linker_Bildrand),_Fahrbahnvorbereitung_am_Nordufer.jpg (not with the Android app), with coordinates.

Problem: The coordinates are not visible when tapping on the picture in "My recent uploads".

Metadata of the file:
```
=={{int:filedesc}}==
{{Information
|description = Neubau der Freiburger Kronenbrücke (linker Bildrand), Fahrbahnvorbereitung am Nordufer
|date = 2018-07-11 13:28
|source = {{own}}
|author = [[User:Andreas Schwarzkopf|Andreas Schwarzkopf]]
|permission = 
|other_versions = 
}}
{{Location dec|47.99081|7.845416|heading:255.9}}

=={{int:license-header}}==
{{cc-by-sa-3.0}}

[[Category:Photographs by Andreas Schwarzkopf]]
[[Category:Taken with Canon PowerShot A650 IS]]
[[Category:Kronenbrücke, Neubau (Freiburg im Breisgau)]]
[[Category:Uploaded with VicuñaUploader]]
```

**Comment by nicolas-raoul** [comment]
That is probably the cause indeed.
We must support all existing formats.

**Comment by ashishkumar468** [comment]
@maskaravivek @nicolas-raoul I'd like to take this up

**Comment by maskaravivek** [comment]
@ashishkumar468 its yours. :)

**Comment by ashishkumar468** [comment]
Thanks @maskaravivek  :-). Can some one let me know if there is a beta version of the website as well, in order to produce the issue I need to first upload an image from the website and then with the same account I need to debug in the android app to reproduce it and add a probable fix

**Comment by maskaravivek** [comment]
> Can some one let me know if there is a beta version of the website as well
Yes. https://commons.wikimedia.beta.wmflabs.org/

> in order to produce the issue I need to first upload an image from the website and then with the same account I need to debug in the android app to reproduce it and add a probable fix
Or maybe you can just search for this image using the explore feature in the app.

**Comment by nicolas-raoul** [comment]
Yes, just use the Explore feature and search for the image's name, you will probably find it easily.
The bug is most probably visible via that activity too.

**Comment by ashishkumar468** [comment]
@nicolas-raoul Yeah, was easier that way, have added the fix, raising the pull request in a while

**Comment by ashishkumar468** [comment]
@maskaravivek @nicolas-raoul Have raised a pull request for the same. It would be great if one of you could review https://github.com/commons-app/apps-android-commons/pull/1845

**Comment by domdomegg** [comment]
Fixed in #1845
